ingestion/functions/parsing/cuba/sample_data.json

```diff
@@ -0,0 +1,244 @@
+{
+    "schema-version": 7,
```

Comment:
interesting that they version their schema, perhaps your parser could look at that and print a message if it's not 7 anymore? Also leave a comment in the parsing code that it was built for v7 of the schema.